prov/efa/src/rxr/rxr_cq.c

```diff
@@ -47,22 +47,14 @@
 static const char *rxr_cq_strerror(struct fid_cq *cq_fid, int prov_errno,
 				   const void *err_data, char *buf, size_t len)
 {
-	struct fid_list_entry *fid_entry;
-	struct util_ep *util_ep;
 	struct util_cq *cq;
-	struct rxr_ep *ep;
 	const char *str;
 
 	cq = container_of(cq_fid, struct util_cq, cq_fid);
 
 	ofi_mutex_lock(&cq->ep_list_lock);
 	assert(!dlist_empty(&cq->ep_list));
-	fid_entry = container_of(cq->ep_list.next,
-				 struct fid_list_entry, entry);
-	util_ep = container_of(fid_entry->fid, struct util_ep, ep_fid.fid);
-	ep = container_of(util_ep, struct rxr_ep, util_ep);
-
-	str = fi_cq_strerror(ep->rdm_cq, prov_errno, err_data, buf, len);
+	str = efa_strerror(prov_errno);
 	ofi_mutex_unlock(&cq->ep_list_lock);
 	return str;
 }
```

Comment:
I was unsure what to do with this function in general. Replacing the `fi_cq_strerror()` call with `efa_strerror()` makes `fid_entry`, `util_ep` and `ep` go unused, so they were removed.